understanding/name-role-value.html

```diff
@@ -58,106 +56,79 @@
                   
                </p>
-               <p>When standard controls from accessible technologies are used, this process is straightforward.
-                  If the user interface elements are used according to specification the conditions
-                  of this provision will be met. (See examples of Success Criterion 4.1.2 below)
+               <p>アクセシブルなウェブコンテンツ技術の標準コントロールを使用する場合、このプロセスは簡単である。ユーザインタフェース要素が仕様に準じて使用される場合、この条件に条項は満たされる。(下記の達成基準 4.1.2 の事例を参照のこと)
                   
                </p>
-               <p>If custom controls are created, however, or interface elements are programmed (in
-                  code or script) to have a different role and/or function than usual, then additional
-                  measures need to be taken to ensure that the controls provide important and appropriate
-                  information
-                  to assistive technologies and allow themselves to be controlled by assistive technologies.
+               <p>しかし、カスタムコントロールが作成される、又はインタフェース要素が通常とは異なる役割及び／又は機能を持つように (ソースコード又はスクリプトで) プログラミングされている場合、コントロールが支援技術に重要及び適切な情報を提供し、支援技術によってコントロールされることを、それら自身ができるように保証する追加の手段を取る必要がある。
                   
                </p>
-               <p>What roles and states are appropriate to convey to assistive technology will depend
-                  on what the control represents. Specifics about such information are defined by other
-                  specifications, such as <a href="https://www.w3.org/TR/wai-aria/">WAI-ARIA</a>, or the
-                  relevant platform standards. Another factor to consider is whether there is sufficient
-                  <a href="https://www.w3.org/TR/WCAG22/#dfn-accessibility-supported">accessibility support</a> 
-                  with assistive technologies to convey the information as specified.
+               <p>どのような役割 (role) 及び状態を支援技術に伝えるのが適切かは、コントロールが何を表すかによる。そのような情報に関する詳細は、<a href="https://www.w3.org/TR/wai-aria/">WAI-ARIA</a> などの他の仕様、関連するプラットフォーム標準によって定義される。考慮すべきもう一つの要素は、指定されたとおりに情報を伝達するために、支援技術とともに十分な<a href="https://waic.jp/translations/WCAG22/#dfn-accessibility-supported">アクセシビリティサポート</a>があるかどうかである。
                   
                </p>
-               <p>A particularly important state of a user interface control is whether or not it has
-                  focus. The focus state of a control can be programmatically determined, and notifications
-                  about change of focus are sent to user agents and assistive technology. Other examples
-                  of user interface control state are whether or not a checkbox or radio button has
-                  been selected, or whether or not a collapsible tree or list node is expanded or collapsed.
+               <p>特に重要なユーザインタフェース コントロールの状態は、フォーカスを持つかどうかである。コントロールのフォーカス状態は、プログラムによる解釈が可能であり、フォーカスの変化に関する通知は、ユーザエージェント及び支援技術に送られる。ユーザインタフェース コントロールのステータスの他の例としては、チェックボックス又はラジオボタンが選択されているかどうか、折り畳み可能なツリーやリストが展開されている又は折り畳まれているかどうかである。
```

Comment:
「他の例としては…である」となっていますが、「他の例は…である」or「他の例としては…がある」かと思います。

```suggestion
               <p>特に重要なユーザインタフェース コントロールの状態は、フォーカスを持つかどうかである。コントロールのフォーカス状態は、プログラムによる解釈が可能であり、フォーカスの変化に関する通知は、ユーザエージェント及び支援技術に送られる。ユーザインタフェース コントロールのステータスの他の例としては、チェックボックス又はラジオボタンが選択されているかどうか、折り畳み可能なツリーやリストが展開されている又は折り畳まれているかどうかがある。
```